javascript/javascript_in_the_real_world/linting.md

```diff
@@ -66,7 +66,7 @@ If you find yourself reusing a lot of setup code for multiple projects, you can
 
 The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
 
-- [What is linting?](https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/)
+- [What is linting?](https://gomakethings.com/javascript-linters/) <!-- previous link gave a 404 error -->
```

Comment:
Thanks for replacing the broken link @rajvatsal. Could you remove the comment? We just need the link href changed.
```suggestion
- [What is linting?](https://gomakethings.com/javascript-linters/)
```